examples/11_melt_native_token.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! cargo run --example 11_melt_native_token --release
+//! cargo run --example 11_decrease_native_token_supply --release
```

Comment:
Change the file name as well ?

Reply:
Ahh, I actually changed it back because melting is shorter and still makes sense :trollface:
So I could update the comment or change the file name, what do you think is less confusing?

Reply:
Since it's user facing I would harmonise everything so that they are not confused if the file is called melting but we never call anything called "melt*"